packages/core/helpers/ver/4.24.9.ts

```diff
@@ -8,10 +9,28 @@ const patchSingleStateZoom = newConfig => {
   }
 }
 
+const patchDashboardFilterActiveStatus = newConfig => {
+  if (!newConfig.dashboard) return newConfig
+  const dashboardConfig = newConfig as MultiDashboardConfig
+
+  // previous filters need an active status
+  const newSharedFilters = (dashboardConfig.dashboard.sharedFilters || []).map(sf => {
+    if (!sf.active && sf.values) {
```

Comment:
this seems to be reverting the changes I made to formatConfigBeforeSave https://github.com/CDCgov/cdc-open-viz/blob/294572787fe1d47e61a4613636adf640ca30fec0/packages/core/helpers/formatConfigBeforeSave.ts#L57

I added this change because the selection of active state is a side effect of the WCMS developer using the dropdowns. It should instead either be set to some default selection on load of the dashboard either through a query parameter if one is set, to an empty value if reset label is given, or otherwise the first available value.